pkg/resources/aws.go

```diff
@@ -701,8 +701,8 @@ func ListKeypairs(cloud fi.Cloud, clusterName string) ([]*Resource, error) {
 
 	glog.V(2).Infof("Listing EC2 Keypairs")
 	request := &ec2.DescribeKeyPairsInput{
-	// We need to match both the name and a prefix
-	//Filters: []*ec2.Filter{awsup.NewEC2Filter("key-name", keypairName)},
+		// We need to match both the name and a prefix
+		//Filters: []*ec2.Filter{awsup.NewEC2Filter("key-name", keypairName)},
```

Comment:
Can you remove this file out of the commit.  This file is where you are having gofmt issue.  In go 1.10 this is correctly formatted.  In go 1.9 no changes are required.  So please revert this file :)